ansible/roles/spdashboard/tasks/spdashboardvm.yml

```diff
@@ -1,11 +1,12 @@
 ---
 - name: Install some packages needed for development
-  yum: pkg={{ item }} state=present
-  with_items:
-    - ruby-devel
-    - gcc-c++
-    - sqlite-devel
-    - ant
+  yum: 
```

Comment:
Nice, this notation is much more readable than the previous 'with_items' solution! I assume we do not have to support prehistoric (pre 2.0) ansible versions?

Reply:
Indeed!